### PR TITLE
Create issue_template.md

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### Hail uses issues to track bugs. For support and feature requests, please use the discussion forum: http://discuss.hail.is/
+
+### Hail version used:
```
Comment (jigold) [inline .github/issue_template.md]: Can you increase the number of "#" on the headers or make them bullets instead? I don't like looking at multiple lines of bold font sequentially.

**Comment by tpoterba** [comment]
back to you

**Comment by jigold** [comment]
Sorry, I should have been clearer. I think the first line shouldn't have any "#"s and for the following lines, "###" is fine.

**Comment by jigold** [comment]
@tpoterba I added a commit. Please modify if not happy with the changes, otherwise, I will approve.

**Comment by tpoterba** [comment]
Looks great!

**Pull request merged**